.github/actions/deploy/action.yaml

```diff
@@ -27,6 +27,9 @@ inputs:
   cdn_path:
     description: 'The CDN bucket S3 URL'
     required: false
+  tenant:
+    description: 'The Tenant inherent to the website'
+    required: false
```

Comment:
why is this field not required? what happen if a caller do not pass the `tenant` as inout?

Reply:
sorry. with last commit i set the parameter "required: true"